### PR TITLE
Move simple-option repo

### PR DESCRIPTION
**Pull request by orkunkl**
Related to https://github.com/CosmWasm/cosmwasm-plus/

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -270,3 +270,54 @@ jobs:
...
+          name: Add wasm32 target
+          command: rustup target add wasm32-unknown-unknown
+      - run:
+          name: Build
+          command: cargo wasm --locked
+      - run:
+          name: Unit tests
+          env: RUST_BACKTRACE=1
+          command: cargo unit-test --locked
+      - run:
```
Comment (ethanfrey) [inline .circleci/config.yml:296]: I believe there are not integration tests here, we can probably remove this line

Comment (ethanfrey) [reply .circleci/config.yml:296]: In fact this is not in cargo, meaning this should fail, meaning these tests are not yet run.

Please add simple-option to the jobs on top, let's see the test failure with this line, then remove that in another commit.

**Review by ethanfrey — APPROVED** [review]
Looks good.

But please fix up the CI setup and ensure all tests pass before mering

File: .circleci/config.yml (hunk 1 above)
Comment (orkunkl) [reply .circleci/config.yml]: I moved it to the top but Circle CIwroker did not see the change.

Comment (orkunkl) [reply .circleci/config.yml]: I found what you mean :+1:

**Comment by ethanfrey** [comment]
Nice

**Pull request merged**